src/js/tech/ChromecastTech.js

```diff
@@ -163,6 +163,8 @@ ChromecastTech = {
       this.trigger('waiting');
       this._clearSessionTimeout();
 
+      mediaInfo.duration = this.duration();
```

Comment:
This sets the duration correctly for the first item that's played, but if the `src` changes (e.g. if someone is playing items one right after another in a playlist of some sort), `this.duration()` will return the `duration` of the previous `src` and `mediaInfo.duration` will be incorrect.

Is this even strictly necessary? It would be nice to be able to set this property, but the Chromecast seems to be able to calculate the duration and play fine without us setting it. We don't know what the `duration` of the `src` is at this point in the case I described above.